src/styles/components/form-elements/form-row/styles.less

```diff
@@ -14,6 +14,11 @@
       margin-bottom: 0;
     }
   }
+
+  // Add top-padding equivalent to a label
+  .form-group-without-top-label {
+    padding-top: 1.82rem;
+  }
```

Comment:
I'd like it if we could construct this value based on the CNVS variables. So that would be as follows: `@label-line-height + @label-margin-bottom`. Let's also make sure to add the breakpoints as necessary.

CNVS doesn't change the label line height for any breakpoint, but it does change the margin for breakpoints `medium` and `large`, so we just need to account for those two.

Line-height variable: https://github.com/mesosphere/cnvs/blob/develop/styles/components/form/label/variables.less#L107
Margin variables: https://github.com/mesosphere/cnvs/blob/develop/styles/components/form/label/variables.less#L129-L133